apps/allocations/app/components/App/App.js

```diff
@@ -134,7 +134,7 @@ const App = () => {
   const PanelContent = panel ? panel.content : null
 
   return (
-    <Main scrollView={false}>
+    <Main scrollView={true}>
```

Comment:
```suggestion
    <Main>
```

As the [this prop](https://github.com/aragon/aragon-ui/tree/newstyle/src/components/Main#scrollview) is already set to `true` by default, it seems confusing to add it, I think it would make sense to just get rid of the prop, we usually don't add props when we use the defaults. Do you think of a good reason to keep it @PeterMPhillips ?

Reply:
It was set to false which was preventing the page for scrolling. If removing it has the same effect than I'll do that.

Reply:
It seems that aragon-ui docs are wrong then! thanks for testing it!